libfreerdp/core/test/TestFuzzCryptoCertificateDataSetPEM.c

```diff
@@ -0,0 +1,16 @@
+#include <stddef.h>
+#include <stdint.h>
+#include <freerdp/crypto/certificate.h>
+
+extern int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size)
+{
+	rdpCertificateData* data = NULL;
+	data = certificate_data_new("somehost", 1234);
+	if (!data)
+		return 0;
+	if (!certificate_data_set_pem(data, (const char *)Data))
```

Comment:
`API` misuse, `certificate_data_set_pem` expects a string (that is, must end with `\0`) which is most likely not the case with the fuzzer input.
Create a copy and add a `\0` at the end